projects/observability/src/pages/apis/backends/backend-list.component.ts

```diff
@@ -41,7 +42,8 @@ export class BackendListComponent {
         value: {
           type: 'attribute-specification',
           attribute: 'type'
-        }
+        },
+        sortable: false
```

Comment:
Why disable sorting?

Reply:
Can you expand the column here? It's for a table which shows diagram in that column, the sorting actually doesn't make sense for it, hence disabled.